### ISSUE TITLE
Could you plz also release the config for 100ep pretraining?

### ISSUE THREAD

**Issue by HuangChiEn** [ZhichengHuang/CMAE#6]
Hello, I'm wondering did you'll have conduct the experiments for 100ep pertaining of CMAE ?
If you have could you release the 100ep pertaining / fine-tuning setup config. (such as lr, layer-decay, ...etc.) (or at least give us some hint about your setup) ?

> Since we have reimplement the CMAE according to the paper (although we applied the BYOL loss), the accuracy seems hard to achieved the accuracy of MAE.
(In our [implementation](https://wandb.ai/josef/MixSim/runs/35ti896x/overview?workspace=user-josef), it only got 79.35% top-1 acc) 
I think it caused by our raw hyper-params config (we use same config as MAE-1600ep pretraining), so that the training config may not match with CMAE case and degrade the performance. 

Any suggestion will be appreciated !!

**Comment by ZhichengHuang** [comment]
I'm sorry to tell you that our minimum experiment setup is running 300 epochs. In this experiment, we followed the hyperparameter settings in MAE, with the same linear lr scaling rule and batch size of 128*32 as in MAE, and a weight decay of 0.05 and warmup of 40 epochs.  For the pre-training setting of 300 epochs, we maintain the same settings as in the MAE paper during fine-tuning.  I hope this information is helpful. Thank you.

**Comment by HuangChiEn** [comment]
> with the same linear lr scaling rule and batch size of 128*32 as in MAE, and a weight decay of 0.05 and warmup of 40 epochs

I see. thanks for your information. It's helpful ~

**Comment by HuangChiEn** [comment]
By the way, if you don't mind could you confirm the 300ep-pretaining / 100ep-fine-tuning config  for CMAE ?
---
300ep pretraining 
--batch_size 4096 \ (confirm!)
--norm_pix_loss \
--mask_ratio 0.75 \
--warmup_epochs 40 \ (confirm!)
--base_learning_rate 1.5e-4 \
--weight_decay 0.05 \ (confirm!)

100ep finetuning
--batch_size 1024 \
--model vit_base_patch16 \
--epochs 100 \
--base_learning_rate 1.5e-4 \
--layer_decay 0.65 \
--weight_decay 0.05 \ (confirm!)
--drop_path 0.1 \
--reprob 0.25 \
--mixup 0.8 \
--cutmix 1.0 \

then, we can close the issue.

**Comment by HuangChiEn** [comment]
Ok, i take no response as all cfg is matched. 

> #### By the way, if you don't mind could you confirm the 300ep-pretaining / 100ep-fine-tuning config for CMAE ?
> 300ep pretraining --batch_size 4096 \ (confirm!) --norm_pix_loss --mask_ratio 0.75 --warmup_epochs 40 \ (confirm!) --base_learning_rate 1.5e-4 --weight_decay 0.05 \ (confirm!)
> 
> 100ep finetuning --batch_size 1024 --model vit_base_patch16 --epochs 100 --base_learning_rate 1.5e-4 --layer_decay 0.65 --weight_decay 0.05 \ (confirm!) --drop_path 0.1 --reprob 0.25 --mixup 0.8 --cutmix 1.0 \
> 
> then, we can close the issue.
> 
> > I'm sorry to tell you that our minimum experiment setup is running 300 epochs. In this experiment, we followed the hyperparameter settings in MAE, with the same linear lr scaling rule and batch size of 128*32 as in MAE, and a weight decay of 0.05 and warmup of 40 epochs. For the pre-training setting of 300 epochs, we maintain the same settings as in the MAE paper during fine-tuning. I hope this information is helpful. Thank you.



> #### By the way, if you don't mind could you confirm the 300ep-pretaining / 100ep-fine-tuning config for CMAE ?
> 300ep pretraining --batch_size 4096 \ (confirm!) --norm_pix_loss --mask_ratio 0.75 --warmup_epochs 40 \ (confirm!) --base_learning_rate 1.5e-4 --weight_decay 0.05 \ (confirm!)
> 
> 100ep finetuning --batch_size 1024 --model vit_base_patch16 --epochs 100 --base_learning_rate 1.5e-4 --layer_decay 0.65 --weight_decay 0.05 \ (confirm!) --drop_path 0.1 --reprob 0.25 --mixup 0.8 --cutmix 1.0 \
> 
> then, we can close the issue.
> 
> > I'm sorry to tell you that our minimum experiment setup is running 300 epochs. In this experiment, we followed the hyperparameter settings in MAE, with the same linear lr scaling rule and batch size of 128*32 as in MAE, and a weight decay of 0.05 and warmup of 40 epochs. For the pre-training setting of 300 epochs, we maintain the same settings as in the MAE paper during fine-tuning. I hope this information is helpful. Thank you.

**Comment by HuangChiEn** [comment]
Ok, i take no response as the config is all matched!!
